Tests/MessagingInApp/APITest.swift

```diff
@@ -19,6 +19,22 @@ class MessagingInAppAPITest: UnitTest {
         try skipRunningTest()
 
         MessagingInApp.shared.initialize(organizationId: "")
-        mock.initialize(organizationId: "")
+        MessagingInApp.shared.initialize(organizationId: "", eventListener: self)
```

Comment:
I think it is good to write test case without eventListener as a parameter or by passing a nil parameter to test the other possibility